caddyconfig/caddyfile/parse.go

```diff
@@ -451,6 +465,12 @@ func (p *parser) doSingleImport(importFile string) ([]Token, error) {
 		return nil, p.Errf("Could not read imported file %s: %v", importFile, err)
 	}
 
+	// only warning in case of empty files
+	if len(input) == 0 {
+		caddy.Log().Warn("import file is empty", zap.String("file", importFile))
+		return []Token{}, nil
+	}
```

Comment:
I think we might need to go one step further here and check if the input is only whitespace (i.e. spaces and/or newlines) or empty.

Also, I think the warning message should start with an uppercase letter, like the other ones in this file (i.e. `"Import file is empty"`).

Reply:
@francislavoie I took care of this.